frontends/ol-util/src/components/Card.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react"
+import classnames from "classnames"
+
+interface CardProps {
+  children: React.ReactNode
+  className?: string
+  title?: React.ReactNode
+  borderless?: boolean
+  persistentShadow?: boolean
+}
+
+/**
+ * Renders child content into some styled divs. Optionally specify whether the
+ * card has border and permanent shadow.
+ */
+const Card: React.FC<CardProps> = ({
+  className,
+  persistentShadow,
+  borderless,
+  title,
+  children,
+}) => {
+  return (
+    <div
+      className={classnames("card", className, {
+        borderless,
+        "persistent-shadow": persistentShadow,
+      })}
```

Comment:
We have a whole bunch of custom `getClassName` logic throughout open-discussions. This is [`classnames`](https://www.npmjs.com/package/classnames), it's great.